bdns/dns.go

```diff
@@ -236,26 +236,26 @@ func NewDNSClientImpl(
 	}
 }
 
-// NewTestDNSClientImpl constructs a new DNS resolver object that utilizes the
+// NewTest constructs a new DNS resolver object that utilizes the
```

Comment:
This function is only called from dns_test.go. Should it just move to that file?

Reply:
It's also called from cmd/boulder-va/main.go and va/dns_test.go.

Reply:
d'oh, dunno why I missed those. LGTM.

Reply:
They're easy to miss! It's a big change with a lot of boilerplate diffs.